README.md

```diff
@@ -87,21 +89,25 @@ This is achieved using AWS CloudFormation. For more details check our [CloudForm
 The controller *will not* manage the security groups required to allow access from the Internet to the load balancers.
 It assumes that their lifecycle is external to the controller itself.
 
+Every poll cycle Kubernetes cluster is scanned for new nodes. Each new
+discovered node is scanned for Auto Scaling Group tag. Each Target
+Group created by this Ingress controller is then added to each Auto
+Scaling Group.
+If instance does not belong to Auto Scaling Group (does not have
+`aws:autoscaling:groupName` tag) it is stored in separate list of
+Single Instances. On each cycle instances on this list are registered
+as tartgets in all Target Groups managed by this controller.
```

Comment:
I think this part fits better into "Discovery", not?

Reply:
I took it out of initial discovery phase and added to each cycle because otherwise newly added instances (after start of Ingress pod) would not be discovered and handled correctly.
I did everything I could to minimize number of calls to AWS APIs though.

Reply:
ok